home1/task0.js

```diff
@@ -0,0 +1,26 @@
+// way 1
+let accumulator = 0;
+function add(value) {
+  if (value) {
+    accumulator += value;
+    return add;
+  } else {
+    return accumulator;
+  }
+}
+
+console.log("way 1: expected value 36. result =", add(2)(5)(7)(1)(6)(5)(10)()); // 36
+
+// way 2
+function add2(value, accumulator = 0) {
```

Comment:
Way 2 is better since it stores sum in closure. So different `add` executions would work well.